pkg/eve/runme.sh

```diff
@@ -246,8 +249,8 @@ ACTION="do_$1"
 [ "$(type -t "$ACTION")" = "$ACTION" ] || bail "Error: unsupported command '$1' - use 'help' command for more information."
 shift
 
-# If /in was provided, for now we assume it was to override configs
-if mountpoint -q /in; then
```

Comment:
If user has mounted a single or multiple files using -v, /in will be a mountpoint, so I'm in favor to keep the checking with `mountpoint`, or there is a particular reason to change to `ls -A` ?

Reply:
The thing is, if you mount single files instead of a folder, each of the files will be it's own mountpoint and the check `mountpoint -q /in` will fail. A use case for mounting files is for example if the files lie in different paths or you don't want the whole folder content to be mounted.
We could check something like 
```sh
mount | grep "/in"
```
but I think `ls -A /in` is simpler and cleaner, since we don't expect any other files to be present there except the mounted ones.

Reply:
Added the explanation to the commit message :+1: